src/components/Link/List.js

```diff
@@ -47,25 +61,34 @@ class LinkList extends Component {
   };
 
   render() {
-    const { children, ...rest } = this.props;
+    const { children, renderAfter, ...rest } = this.props;
 
     return (
       <LinkListProvider value={this.state}>
-        <Container {...rest}>{children}</Container>
+        <Container {...rest}>
+          <Wrapper
+            className={classNames({ "wrapper--has-after": !!renderAfter })}
+          >
+            {children}
+          </Wrapper>
+          {renderAfter && <AfterWrapper>{renderAfter}</AfterWrapper>}
```

Comment:
I feel like `renderAfter` is ambiguous... how do you feel about renaming it to `containerChildren`?

Reply:
Also, can you please add the new prop to the catalog documentation?